lib/Dialect/SPIRV/Serialization/Deserializer.cpp

```diff
@@ -722,6 +749,46 @@ LogicalResult Deserializer::processFunctionType(ArrayRef<uint32_t> operands) {
   return success();
 }
 
+LogicalResult Deserializer::processStructType(ArrayRef<uint32_t> operands) {
+  if (operands.size() < 2) {
+    return emitError(unknownLoc, "OpTypeStruct must have at least one member");
```

Comment:
Actually the spec explicitly says "Declare a new structure type: an aggregate of zero or more potentially heterogeneous members." So we can have zero members.

Reply:
Thanks for mention this, but it seems like the current spv.struct must have at least one field, sorry if I'm missing something.

Reply:
Maybe that is the case. Have to look into it.